prow/config/config.yaml

```diff
@@ -461,6 +461,22 @@ presubmits:
           value: "TRUE"
         image: docker.io/golang:1.18
         imagePullPolicy: Always
+  - name: golangci-lint
+    branches:
+    - main
+    skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/ensure-golangci-lint.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: docker.io/golang:1.19
+        imagePullPolicy: Always
```

Comment:
this is a duplicate of lines 448-463 with correct go version (1.19 for main branch), so remove lines 448-463

Reply:
Yes I screwed it while rebasing, working on it, actually I have to do the other way around, keep it and remove 448-463 since that has go 1.18.

Reply:
That is what I suggested as well I guess 😀

Reply:
Ah true :D